### PR TITLE
WIP Rahti4 migration

### PR DESCRIPTION
**Pull request by lvarin**
https://csc-guide-preview.rahtiapp.fi/origin/Rahti4-migration/

### CODE REVIEW

File: docs/cloud/moving-into.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+features needed for web based applications
+* Complex prepackaged application like Apache Spark. It’s also easy to scale up applications for multiple
+users.
+* Deploying from a catalog with many commonly used application templates with just one command.
+
+In other words, in cases where scalability is needed or you only want to manage a single application, container
+orchestration service might be the way to go.
+
+Containers are inherently not as secure as Virtual machines. With container orchestration services like
+Rahti and Openshift OKD containers aren’t allowed to run with root access, which may provide  some limitations
```
Comment (lvarin) [inline docs/cloud/moving-into.md:40]: This is a minor style thing. In written form "are not" is better than "aren't".

File: docs/cloud/moving-into.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+in certain circumstances. For sensitive data processing tasks ePouta should always be the preferred platform. It
+is specifically designed for high performance and high security tasks for sensitive data handling with variety
+of available resources.
+
+### Additional information
+
+[What is Rahti?](https://docs.csc.fi/cloud/rahti/rahti-what-is/)
```
Comment (lvarin) [inline docs/cloud/moving-into.md:47]: Internal links to docs.csc.fi should be relative links and cannot contain `https://docs.csc.fi/`

File: docs/cloud/moving-into.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+Computing platforms, but there are some important differences. Virtual machine based services, like
+cPouta, are classified as Infrastructure-as-a-Service (IAAS) cloud services. Kubernetes on the other
+hand provides a container cloud orchestration service. You can deploy services just by using
+Kubernetes, but typically it provides building blocks for developer platforms such as Rahti or
+OpenShift OKD, which are a Platform-as-a-Service (PAAS) systems.
+
+The primary difference between IAAS and PAAS services is that with IAAS the user is responsible
+for managing the operating system and potentially multiple applications whereas with PAAS, the
+user is only responsible for managing the application and the data. You can read more about cloud
+computing concepts [here](https://docs.csc.fi/cloud/concepts/).
```
Comment (lvarin) [inline docs/cloud/moving-into.md:14]:
```suggestion
[computing concepts](../concepts/).
```
Minor thing, internal links to docs.csc.fi should be relative links and cannot contain `https://docs.csc.fi/`. Also links should avoid general words like `here`. (I guess I have done this before as well, but little by little we can improve).

File: docs/cloud/moving-into.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+## When to migrate to K8s
+
+When considering migrating your services from Pouta to k8s, it’s important to consider the relative
+strengths of each platform. Both Pouta and Kubernetes-based platforms, such as Rahti, are Cloud
+Computing platforms, but there are some important differences. Virtual machine based services, like
+cPouta, are classified as Infrastructure-as-a-Service (IAAS) cloud services. Kubernetes on the other
```
Comment (lvarin) [inline docs/cloud/moving-into.md:6]: Do we have any article to link for IAAS and PAAS further study? Internal or external like the wikipedia?

**Review by jemaltahir — COMMENTED** [review]
:)

**Comment by trispera** [comment]
Merged into `cloud-redone`
https://csc-guide-preview.rahtiapp.fi/origin/cloud-redone
PR: https://github.com/CSCfi/csc-user-guide/pull/1833

**Pull request closed without merging**